pandas/tseries/tests/test_timeseries.py

```diff
@@ -2362,7 +2362,7 @@ def test_map(self):
         f = lambda x: x.strftime('%Y%m%d')
         result = rng.map(f)
         exp = [f(x) for x in rng]
-        self.assert_numpy_array_equal(result, exp)
+        np.testing.assert_array_equal(result, exp)
```

Comment:
@jreback I changed it because `assert_numpy_array_equal()` actually would fail in some of the testing environments.  It seems like a problem with the older `numpy` version (1.8.x)

what should I change it to instead?